### PR TITLE
Adding Email validation

### PR DESCRIPTION
**Pull request by SnakyMz**
Things we have done:

- [ ] Adding Error message 'small'
- [ ] Adding Error message styling
- [ ] Adding Error message JavaScript

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -191,3 +191,16 @@ const generateProjects = () => {
...
+const email = document.querySelector('#email');
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  errorMail.innerText = '';
+  if (/[a-z]/.test(email.value) && /[A-Z]/.test(email.value)) {
+    errorMail.innerText = 'Please a provide valid lowercase email address';
+  } else {
+    form.submit();
+  }
+});
```
Comment (Ofarouq310) [inline script.js:206]: - [ ] Every requirement and form validation condition has been implemented correctly, but kindly note that if your form is ready for submission and you click on the button, it does not work, and throws that error, kindly address this issue. 
  
 image

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @SnakyMz & @obibaadoma, ✋ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

giphy

## Highlights
✅ Followed GitHub flow.
✅ Meaningful commit messages.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (SnakyMz) [reply script.js:206]: Please any advice why this could be happening?

Comment (Ofarouq310) [reply script.js:206]: Hello @SnakyMz,
When you give your `form button` and `id` of "submit", you kind of like override the submit() function on this code, so simply put, change the `id` name of your `form button`.

Comment (SnakyMz) [reply script.js:206]: Thank you very much. I have learned something new thanks to you. Ill make changes rightaway

Comment (Ofarouq310) [reply script.js:206]: I wish you all the best, you have what it takes.

**Review by Ofarouq310 — APPROVED** [review]
Hi @SnakyMz & @obibaadoma , ✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇 

giphy

## Highlights
✅ Milestone works as expected.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (SnakyMz) [reply script.js:206]: Thank you very much. Best of Luck to you too. May you have prosperous life up ahead.

**Pull request merged**